aws/templates/common.ftl

```diff
@@ -1245,6 +1254,27 @@
             ]
             [#break]
 
+        [#case "s3"]
+            [#local id =
+                formatComponentS3Id(
+                    target.Tier,
+                    target.Component,
+                    target)]
+            [#local result +=
+                {
+                    "ResourceId" : id,
+                    "Attributes" : {
+                        "NAME" : getExistingReference(id, NAME_ATTRIBUTE_TYPE),
+                        "FQDN" : getExistingReference(id, DNS_ATTRIBUTE_TYPE),
+                        "INTERNAL_FQDN" : getExistingReference(id, DNS_ATTRIBUTE_TYPE),
+                        "WEBSITE_URL" : getExistingReference(id, URL_ATTRIBUTE_TYPE),
+                        "ARN" : getExistingReference(id, ARN_ATTRIBUTE_TYPE),
+                        "REGION" : regionId
```

Comment:
Is the regionId based on the set context of the current run? for cross region components like S3, should this be part of the getExistingReference?